### PR TITLE
Added ISO 3166-1 status property

### LINKED ISSUE #253: Bug: XK (Kosovo) is not an officially assigned code element
According to [ISO 3166-1 alpha-2](https://www.iso.org/obp/ui/#iso:pub:PUB500001:en) XK is a 'User-assigned code element'. Kosovo does not have a ISO country code (without being political).

I recommend either removing the entry (thus sticking with ISO 3166-1) or extending the data set to include non compliant entries.

[Explanation](https://www.quora.com/Why-doesnt-Kosovo-have-its-own-ISO-country-code-when-it-is-already-eligible-for-it-it-is-member-of-IMF-and-UN-treats-it-as-a-territory-separate-from-Serbia)
 

**Proposal:**
a) remove entry
_or_
b) add "status" flag to every record:

| status |
| --- |
| officially assigned |
| unassigned |
| exceptionally reserved |
| indeterminately reserved |
| transitionally reserved |
| user-assigned |
| formerly used |

This change would make the data set compliant with ISO 3166.

[see Glossary ISO 3166](https://www.iso.org/glossary-for-iso-3166.html)
or [Search filter](https://www.iso.org/obp/ui/#search)

### PR DESCRIPTION
**Pull request by blumk**
Added status property (fixes #253). This property is part of [ISO 3166-1 ](https://www.iso.org/obp/ui/#iso:pub:PUB500001:en).

Added "officially-assigned" to 247 (194 + 53) records, added "user-assigned" to one record (Kosovo).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ This repository contains lists of world countries in JSON, CSV and XML. Each lin
  - code ISO 3166-1 numeric (`ccn3`)
  - code ISO 3166-1 alpha-3 (`cca3`)
  - code International Olympic Committee (`cioc`)
+ - independence status from ISO 3166-1 (`independent`) (denotes the country is considered a sovereign state)
+ - ISO 3166-1 status (`status`)
```
Comment (mledoze) [inline README.md]: I propose to change this to: `ISO 3166-1 assignment status`, what you think @blumk?

Comment (blumk) [reply README.md]: @mledoze Sounds good. Will update it today.

Comment (mledoze) [reply README.md]: Thanks.

Comment (blumk) [reply README.md]: @mledoze Done.

File: README.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ This repository contains lists of world countries in JSON, CSV and XML. Each lin
  - code ISO 3166-1 numeric (`ccn3`)
  - code ISO 3166-1 alpha-3 (`cca3`)
  - code International Olympic Committee (`cioc`)
+ - independence status from ISO 3166-1 (`independent`) (denotes the country is considered a sovereign state)
```
Comment (mledoze) [inline README.md]: @blumk I'm sorry but I would like to make another change; for consistency, could you please change the beginning of this line to `ISO 3166-1 independence status`?

Comment (blumk) [reply README.md]: @mledoze No problem, I've just updated the file

Comment (mledoze) [reply README.md]: Thank you. I'm waiting for the Travis build to pass and then I'll merge it.

**Comment by mledoze** [comment]
All is good. Laaaaanding :)

**Pull request merged**